dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/resources/docker/workflow-http/mockserver-config.json

```diff
@@ -0,0 +1,22 @@
+
+[{
+  "httpRequest": {
+    "path": "/test",
+    "method": "GET"
+  },
+  "httpResponse": {
+    "body": [
+      {
+        "name": "lzl",
+        "age": 18
+      },
+      {
+        "name": "pizz2",
+        "age": 19
+      }
+    ],
+    "headers": {
+      "Content-Type": "application/json"
+    }
+  }
+}]
```

Comment:
The last line of each file should be blank.

Reply:
done PTAL @SbloodyS  thks